roku/src/assets/data/content.json

```diff
@@ -0,0 +1 @@
+{"ipAddress":"192.168.2.12"}
```

Comment:
This should work for now 👍  Someone might just need to update this based on their local IP that gets generated when running `npm start` from `.content`